scss/atoms/notifications/_mixins.scss

```diff
@@ -0,0 +1,18 @@
+/*
+* ==========================================================================
+* File: _mixins.scss
+* For : Notifications
+* ==========================================================================
+*/
+
+@mixin notification($type) {
+	border-radius: map-get($notifications-element, border-radius);
+	box-shadow: map-get($notifications-shadows, box-shadow-base);
+	margin: map-get($notifications-element, margin);
+	letter-spacing: map-get($notifications-element, letter-spacing);
+	font-weight: map-get($notifications-element, font-weight);
+	padding: map-get($notifications-element, padding);
+	font-size: map-get($notifications-element, font-size);
+	background: bg($type);
```

Comment:
You don't need to call functions to get this. Check out how we do it in buttons, just pass in the $notification -success/$notifications-default/etc variable into this mixin and map-get the needed variable from that.  Will make it easier if additional notification colors ever get added, instead of having to add an additional else if statement in both of your function files